src/ParallelAlgorithms/LinearSolver/ConjugateGradient/ConjugateGradient.hpp

```diff
@@ -136,7 +140,7 @@ struct ConjugateGradient {
    *   * `LinearSolver::Tags::IterationId`
    *   * `Tags::Next<LinearSolver::Tags::IterationId>`
    */
-  using prepare_step = cg_detail::PrepareStep;
+  using prepare_step = cg_detail::PrepareStep<FieldsTag,OptionsGroup>;
```

Comment:
I think the lack of space after the comma means this file should be run through `clang-format` again.

Reply:
`clang-formatted` everything and squashed the changes in their respective commits directly